product_customer_code/models/product.py

```diff
@@ -0,0 +1,29 @@
+# coding: utf-8
+
+from openerp import api, fields, models
+
+
+class ProductProduct(models.Model):
+
+    _inherit = "product.product"
+
+    product_customer_code_ids = fields.One2many(
+        'product.customer.code', 'product_id', string='Product customer codes',
+        copy=False, help='Customer Codes')
```

Comment:
Don't use the same text of string for help (better let empty help or add a better long help)